lib/instance.js

```diff
@@ -703,11 +703,12 @@ instance.prototype.connect = function (client) {
 				if (typeof field.isVisible === 'function') {
```

Comment:
could this use the implementation of `instance.prototype.serializeIsVisibleFn`? That looks accessible from here

Reply:
Oh yeah! Forgot to use it here. That's the reason I added it to the class as opposed to making it a loose function.